pkg/database/token.go

```diff
@@ -208,7 +208,7 @@ func (db *Database) ClaimToken(authApp *AuthorizedApp, tokenID string, subject *
 // The verCode can be the "short code" or the "long code" which impacts expiry time.
 //
 // The long term token can be used later to sign keys when they are submitted.
-func (db *Database) VerifyCodeAndIssueToken(authApp *AuthorizedApp, verCode string, acceptTypes api.AcceptTypes, expireAfter time.Duration) (*Token, error) {
+func (db *Database) VerifyCodeAndIssueToken(t time.Time, authApp *AuthorizedApp, verCode string, acceptTypes api.AcceptTypes, expireAfter time.Duration) (*Token, error) {
```

Comment:
I think this function should defensively put t into UTC

Reply:
Done everywhere